README.md

```diff
@@ -43,12 +43,19 @@ are supported:
 
 ```javascript
 WebVRConfig = {
+
+  // Flag to disabled the UI in VR Mode
+  CARDBOARD_UI_DISABLED: false, // Default: false
```

Comment:
IMO, it should be `CARDBOARD_UI_ENABLED` and just make sure it's set in `src/main`